libs/SalesforceSDK/src/com/salesforce/androidsdk/security/PasscodeManager.java

```diff
@@ -511,22 +519,23 @@ public void setMinPasscodeLength(int minPasscodeLength) {
              * triggered later from OAuthWebviewHelper.
              */
             if (hasStoredPasscode(SalesforceSDKManager.getInstance().getAppContext())) {
```

Comment:
This one and `storeMobilePolicy` did not really _need_ the Activity context, so I had just left it with the app context. I'll move it over, no problem.